src/components/EventEdit.tsx

```diff
@@ -69,7 +71,6 @@ class EventEdit extends React.PureComponent<PropsType> {
       location: '',
       description: '',
       timezone: null,
-
```

Comment:
Same comment about the space. Again, it's terrible, and will be changed when this component is refactored, though this space keeps some sanity for now.